apps/rehabstod/src/hooks/useSession.ts

```diff
@@ -1,21 +1,28 @@
-import { useEffect } from 'react'
+import { useEffect, useState } from 'react'
 import { useGetSessionPingQuery, useGetUserQuery, useGiveConsentMutation } from '../store/api'
 import { useLogout } from './useLogout'
 
 export function useSession() {
   const { logout } = useLogout()
   const [giveConsent, { isUninitialized }] = useGiveConsentMutation()
+  const [isPollingActive, setIsPollingActive] = useState(true)
   const { data: session, isLoading: isLoadingSession } = useGetSessionPingQuery(undefined, {
     pollingInterval: 30e3,
+    skip: !isPollingActive,
   })
   const { data: user, isLoading: isLoadingUser } = useGetUserQuery()
 
   useEffect(() => {
     if (user && user.pdlConsentGiven === false && isUninitialized) {
       giveConsent({ pdlConsentGiven: true })
     }
-    if (user && session && !session.authenticated) {
-      logout()
+    if (session) {
```

Comment:
`user` from `useGetUserQuery` will always be undefined when `session.authenticate` is false, so we can not look at the user object to determine if logout should be called.

Reply:
This will open ``` window.open('/saml/logout', '_self')``` when the user opens rehabstod, since they are not authenticated yet.